Imagine/Filter/PostProcessor/MozJpegPostProcessor.php

```diff
@@ -88,8 +106,12 @@ public function processWithConfiguration(BinaryInterface $binary, array $options
         $pb = new ProcessBuilder(array($this->mozjpegBin));
 
         // Places emphasis on DC
-        $pb->add('-quant-table');
-        $pb->add(2);
+        $quantTable = array_key_exists('quant_table', $options) ? $options['quant_table'] : $this->quantTable;
+        if (null !== $quantTable) {
```

Comment:
I suppose that this should never be null, if we're reliant on the value of 2 above?  If we were to arrive in that situation, this would be a BC break as we'd no longer be specifying it.